docs/2-techdoc/62304/software-architecture-description.md

```diff
@@ -6,10 +16,20 @@ IEC 62304, para. 5.3.1 and 5.3.2 [class B, C]
 
 **Relevant other documentation:**
 
-* SOP Software Development
-* User needs / stakeholder requirements
-* Design input / software requirements
-* (...)
+- SOP Software Development
+- User needs / stakeholder requirements
+- Design input / software requirements
+- (...)
+
+<!--
```

Comment:
this is duplicate, and probably needs to be deleted

Reply:
Yes, sorry about that. I should have found them before submitting for review. Thanks @sjswerdloff.

Reply:
Fixed in b91dc23